tools/certissuer/lego.go

```
@@ -0,0 +1,134 @@
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+type CertIssuer interface {
+}
+
+type certIssuer struct {
+	ctx context.Context
+	cl  *lego.Client
+}
+
+func NewLego(ctx context.Context, log log.Logger, cfg Config) (CertIssuer, error) {
```

golangci-lint (1.64.8)

[warning] unused-parameter: parameter 'ctx' seems to be unused, consider removing or renaming it as _

(revive)